common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptService.java

```diff
@@ -41,19 +41,43 @@ public ConceptService(EntityManager entityManager) {
   public Slice<Concept> searchConcepts(String query,
       StandardConceptFilter standardConceptFilter, List<String> vocabularyIds,
       List<String> domainIds, int limit) {
+
+      //String[] keywords=query.split("\\s*(=>|,|\\s)\\s*");
+      String[] keywords=query.split("[,+\\s+]");
```

Comment:
Style nit: please add spaces between variable names and operators... e.g. keywords = query.split(...), query2 = new String(), for (String key: keywords), etc.